BinPAC.cmake

```diff
@@ -15,6 +15,12 @@ macro(BINPAC_TARGET pacFile)
         endif ()
 
         set(binpacDep "${BinPAC_EXE}")
+
+        # Ensure that for plugins included via --include-plugins, the Zeek's
+        # source tree paths are added to binpac's include path as well.
+        if ( NOT CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR )
+            set(BinPAC_addl_args "-I;${CMAKE_SOURCE_DIR};-I;${CMAKE_SOURCE_DIR}/src")
```

Comment:
Do we need to add the paths inside the build directory as well for auto-generated files?